.github/workflows/pull_request.yml

```diff
@@ -42,6 +50,25 @@ jobs:
       - name: Install Node.js packages
         if: ${{ steps.modules-cache.outputs.cache-hit != 'true' }}
         run: yarn --frozen-lockfile
+      - name: Set up git config
+        if: ${{github.actor == 'dependabot[bot]' }}
+        run: |
+          git config user.name "${GITHUB_ACTOR}"
+          git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+          git remote set-url origin https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Update manifest
+        if: ${{github.actor == 'dependabot[bot]' }}
+        run: node scripts/createManifest.js
+      - name: Push koku-manifest changes
+        if: ${{github.actor == 'dependabot[bot]' }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git add .
+          git commit --message "Update koku-manifest file"
+          git push
```

Comment:
Could we please move the new actions to a separate "manifest" workflow and change pull_request to run like so?

```
# Trigger when the manifest workflow succeeded
on:
  workflow_run:
    workflows: ["manifest build"]
    types:
      - completed
```

I'm thinking a separate workflow would be cleaner because its a dependabot only feature. It may also be easier to debug and/or disable if needed?

Note that we're migrating to the Insights container build pipeline shortly, so we won't use "pull_request" workflow to build PRs. However, it's possible that we could continue to using a separate "manifest" workflow, while disabling just the "pull_request" workflow.